.github/workflows/lint.yaml

```diff
@@ -4,14 +4,10 @@ on:
   push:
     branches:
       - main
-    paths-ignore:
-      - "**.md"
 
   pull_request:
     branches:
       - main
-    paths-ignore:
-      - "**.md"
```

Comment:
Are you fine with that? I cannot see a problem with lines 11,12. Removing the paths-ignore seems to fix the problem for this PR.

Reply:
you are of course right !

Reply:
however, I do not find this logical, if there is nothing to check, then the action should not block , but report OK

Reply:
I agree, I do not understand how `paths-ignore` is supposed to make sense when it leads to the PR being blocked...